### ISSUE TITLE
Break-Listing on D/M rather than full Datetime

### ISSUE THREAD

**Issue by 4217th** [GothenburgBitFactory/taskwarrior#2062]
Currently report.myreport.sort=due+/ gives listing breaks on datetimes that have any differences at all. 
I had hoped report.myreport.sort=due.day+/ or similar approaches would give a more useful listing (eg per day).

Since only tasks due at the _exact same datetime_ are grouped together and all other are broken up the resulting format is basically having blocks of tasks that were created to be due the same day but without due time (and thus have 00:00:00) and pretty much every other task.

If it's not feasible to give options  for break-listing the result of due+-/ should be braks on day or week

**Comment by pbeckingham** [comment]
Good suggestion.

**Comment by JonasDralle** [comment]
yeah you're right. This is only useful for dates when some sensible default (e. g. days) is provided

**Comment by Fandekasp** [comment]
I would love this feature too. I was trying to create a custom report to display a week worth of tasks following the 1/3/5 methodology, but since I'm adding hours to my due dates, the `rc.report.list.sort:due+/,urgency-` doesn't work as expected right now.
